### DISCUSSION TITLE
GA503RM Troubleshooting BIOS 315/BIOS 313.

### DISCUSSION THREAD

**Discussion by Maneeha1234SL** [seerge/g-helper#1290] [General]
By disabling Fast Startup in Control Panel\Hardware and Sound\Power Options\System Settings,
The problem with the ROG logo appearing twice at startup and
Problems occurring while switching to Egpu will be resolved. (eg #878
(dGPU cannot turn off while using eco mode)

BIOS 315
This method solves these problems in BIOS 313

Grateful. About running a project like this.
Serge❤️

<img width="1280" alt="image" src="https://github.com/seerge/g-helper/assets/83775004/3e49817c-e1cc-44f9-bd85-ec5cc26971e0">

**Comment by 123dqwkhjk** [comment]
Yes, I can confirm it on my ga503rm bios 315.

**Reply by Maneeha1234SL** [reply]
Hello, can you check if disabling this service solves your computer problems?

**Reply by 123dqwkhjk** [reply]
It is worked on my device so far.
